src/llm/http_llm_calculator.cc

```diff
@@ -472,6 +512,17 @@ class HttpLLMCalculator : public CalculatorBase {
             }
             // LOG(INFO) << "Input prompt:" << templateApplyOutput;
 
+            std::string prompt;
+            if (endpoint == CHAT_COMPLETIONS) {
+                RET_CHECK(this->request->getMessages().size() >= 1);
+                RET_CHECK(this->request->getMessages()[0].count("content") >= 1);
+                prompt = this->request->getMessages()[0]["content"];
+            } else if (endpoint == COMPLETIONS) {
+                RET_CHECK(this->request->getPrompt() != "");
+                prompt = this->request->getPrompt();
+            } else {
+                RET_CHECK(false);
+            }
```

Comment:
This doesn't work - prompt is not used and chat template is not taken into account